source/slang/options.cpp

```diff
@@ -859,14 +859,26 @@ struct OptionsParser
 
 
 SlangResult parseOptions(
-    SlangCompileRequest*    compileRequest,
+    SlangCompileRequest*    compileRequestIn,
```

Comment:
Minor point: my convention is to use an `inFoo` prefix style rather than `fooIn` for stuff like this. I also use that convention for cases where a buffer or object is passed by pointer or reference and it seems important to specify whether it is pure input (`in`) pure output (`out`) or input/output (`io`). Abusing that notation to also cover the case where you really want a local variable with the same name as a parameter is kind of gross, but situations like this come up often enough to justify having some kind of policy.